.aws/task-definition-prod.json

```diff
@@ -63,6 +63,10 @@
           "name": "RABBITMQ_USERNAME",
           "valueFrom": "tis-revalidation-rabbit-prod-username"
         },
+        {
+          "name": "RABBITMQ_USE_SSL",
+          "valueFrom": "tis-revalidation-rabbit-use-ssl"
```

Comment:
this may need to match with the param name you set in TIS-OPS `tis-revalidation-rabbit-prod-use-ssl`